packages/tools/devtools/devtools-core/package.json

```diff
@@ -40,8 +82,10 @@
 		"rebuild": "npm run clean && npm run build",
 		"test": "npm run test:mocha",
 		"test:coverage": "c8 npm test",
-		"test:mocha": "mocha \"dist/**/*.test.js\" --exit",
-		"tsc": "tsc",
+		"test:mocha": "npm run test:mocha:esm && echo skipping cjs to avoid overhead - npm run test:mocha:cjs",
```

Comment:
c8 coverage settings should be adjusted - need to be when switching to ESM test only